src/core/functions/aggregate/llm_min_or_max.cpp

```diff
@@ -0,0 +1,210 @@
+#include <flockmtl/core/functions/aggregate/llm_min_or_max.hpp>
+#include <flockmtl/core/functions/prompt_builder.hpp>
+#include <templates/llm_min_or_max_prompt_template.hpp>
+#include "flockmtl/core/module.hpp"
+#include "flockmtl/core/model_manager/model_manager.hpp"
+#include "flockmtl/core/model_manager/tiktoken.hpp"
+
+namespace flockmtl {
+namespace core {
+
+void LlmMinOrMaxState::Initialize() {
+}
+
+void LlmMinOrMaxState::Update(const nlohmann::json &input) {
+    value.push_back(input);
+}
+
+void LlmMinOrMaxState::Combine(const LlmMinOrMaxState &source) {
+    value = std::move(source.value);
+}
+
+LlmMinOrMax::LlmMinOrMax(std::string &model, int model_context_size, std::string &user_prompt,
+                         std::string &llm_min_or_max_template)
+    : model(model), model_context_size(model_context_size), user_prompt(user_prompt),
+      llm_min_or_max_template(llm_min_or_max_template) {
+
+    int fixed_tokens = calculateFixedTokens();
```

Comment:
int -> auto. You should remove the function as it is used in a single place.

`fixed_tokens` -> `num_tokens_meta_and_user_prompts`

Reply:
done